Makefile.docker

```diff
@@ -7,6 +7,7 @@
 # the registry where the image will be pushed (default is Docker Hub).
 TOPDIR=$(dir $(lastword $(MAKEFILE_LIST)))
 DOCKERFILE_DIR     ?= ./
+DOCKER             ?= docker
```

Comment:
Should we rather call it something like `DOCKER_RUNTIME`? Or add some explanatory comment. This seems a bit confusing when it is right next tot he `DOCKER_` vars for configuring the registry.

Reply:
Fair point. Do you think `DOCKER_CMD` would be a bit more self-explanatory?

Reply:
Yeah, it would. Or as I said, maybe just add explanatory comment here.I think in the commands it is more or less clear what it does. It is just here where i is confusing what is the function of it.

Reply:
I used `DOCKER_CMD` because otherwise there are potentially multiple places where a comment might be needed.